examples/platform/linux/CommissionerMain.cpp

```diff
@@ -187,6 +188,12 @@ CHIP_ERROR InitCommissioner(uint16_t commissionerPort, uint16_t udcListenPort)
     gCommissionerDiscoveryController.SetUserDirectedCommissioningServer(gCommissioner.GetUserDirectedCommissioningServer());
     gCommissionerDiscoveryController.SetCommissionerCallback(&gCommissionerCallback);
 
+    // re-init server's fabric table since the CommissionerDiscoveryController will add its fabric if not already there.
```

Comment:
This part is not very clear: is CommissionerDiscoveryController adding the fabric to the _storage_ such that we now need to re-read the storage?  Or is it adding it to the table and we want to ignore that addition?

In general, though, the fabric table doesn't claim that you can call Init on it more than once, so doing that is very odd.    In particular doing this will in-place overwrite data structures that various code (include gCommissioner above) has pointers to, as far as I can tell.

Reply:
added comment explaining rationale, since removed line